docs/databags.md

```diff
@@ -84,6 +84,7 @@ An Indexes Hash is part of a plaintext data bag item that defines the set of ind
     * `noGeneratePaths` - Do not generate the homePath,coldPath,thawedPath to this index when not present in the config above
     * `noRepFactor` - Do not add 'repFactor = auto' to this index when not present in the config on a cluster master.
 * `['metadata']` - These define ownership / other reference metadata around indexes and their owners (ALPHA!!! CAN CHANGE!!!)
+* '_noGenerateTstatsHomePath' - for smartstore enabled indexes tstatsHomePath should be unset.If _noGenerateTstatsHomePath is true in default stanza it means smartstore enabled on all the indexes and on individual indexes smart store can be enabled for per Index. Default is false/nil.
```

Comment:
This should go under `configs`. Also, the description kind of sends the message that setting this attribute will enable smart store. This attribute will only let you not set the tstatshomepath and enabling smart store will require configuring other attributes. How about something like
> Do not generate tstatsHomePath for this index when set to true.

Reply:
updated the comment and moved under config. Thanks